packages/slate/test/interfaces/Operation/inverse/moveNode/ends-before-parent-to-child.js

```diff
@@ -0,0 +1,9 @@
+import { Operation } from 'slate'
+
+export const input = { type: 'move_node', path: [0, 1], newPath: [0, 2, 1] }
+
+export const test = value => {
+  return Operation.inverse(value)
+}
+
+export const output = { type: 'move_node', path: [0, 1, 1], newPath: [0, 1] }
```

Comment:
should the path mutate here?

Reply:
Yes, when the node moving is removed from its starting location `[0, 1]` the node `[0, 2]` slides backwards to fill the gap.